rules/dcos-storage.yml

```diff
@@ -18,4 +18,27 @@ groups:
       service: dcos-storage
     annotations:
       summary: DC/OS storage service has problem connecting to master
-
+  - alert: "Volume provider RECOVERY"
+    expr: count(dss_obj_providers_recovery) > 0
+    for: 1m
```

Comment:
This time is too short to be useful; let's just remove the `for` clause.

Also below.